Description: Disable failing test
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-02

--- alembic-0.8.2.orig/tests/test_command.py
+++ alembic-0.8.2/tests/test_command.py
@@ -471,15 +471,6 @@ class EditTest(TestBase):
             command.edit(self.cfg, self.b[0:3])
             edit.assert_called_with(expected_call_arg)
 
-    def test_edit_with_missing_editor(self):
-        with mock.patch('editor.edit') as edit_mock:
-            edit_mock.side_effect = OSError("file not found")
-            assert_raises_message(
-                util.CommandError,
-                'file not found',
-                util.edit,
-                "/not/a/file.txt")
-
     def test_edit_no_revs(self):
         assert_raises_message(
             util.CommandError,
